code/game/machinery/doors/airlock.dm

```diff
@@ -717,6 +730,10 @@
 
 /obj/machinery/door/airlock/examine(mob/user)
 	. = ..()
+	if(closeOtherId)
+		. += "<span_class='warning'>This airlock cycles on ID: [sanitize(closeOtherId)].</span>"
+	else if(!closeOtherId)
+		. += "<span class='warning'>This airlock does not cycle.</span>"
```

Comment:
I think its fine to remove this, and just have the cycle message

Reply:
if you say so

Reply:
Removed in 24a509c